scripts/cell_simulation.py

```diff
@@ -28,6 +28,21 @@
 import pyggdrasil.tree_inference as tree_inf
 
 
+class NpEncoder(json.JSONEncoder):
+    """Encoder for numpy types."""
+
+    def default(self, obj):
+        """Default encoder."""
+        if isinstance(obj, np.integer):
+            return int(obj)
+        if isinstance(obj, np.floating):
+            # 👇️ alternatively use str()
+            return float(obj)
+        if isinstance(obj, np.ndarray):
+            return obj.tolist()
+        return json.JSONEncoder.default(self, obj)
+
+
```

Comment:
The JSON writer broke down. - don't know why. Might have changed the python version locally. "TypeError: Object of type int64 is not JSON serializable" Implemented this workaround from the web.